docs/contracts.rst

```diff
@@ -1046,8 +1049,8 @@ do it is more convenient if the constructor argument is a
 constant and defines the behaviour of the contract or
 describes it. The second way has to be used if the
 constructor arguments of the base depend on those of the
-derived contract. If, as in this silly example, both places
-are used, the modifier-style argument takes precedence.
+derived contract. If arguments are given in the inheritance list,
```

Comment:
Could you keep the word "modifier-style" (or something similar), I think that helps understanding.

Reply:
Like this?

Reply:
Ok, sorry for nagging again, but perhaps we should not even use the word "overriding" because it suggests that it is possible in some circumstances. Perhaps something like "have to be given either in the inheritance list or in modifier-style in the derived constuctor".

Reply:
Ok, "arguments have to be given either in the inheritance list or in modifier-style in the derived constuctor" sounds like it is invalid not to specify arguments at all, though...